eclipse/sw360
# feat(ProjectTodo): Remove ProjectTodo and UI changes for Obligation and ProjectTodo

> - Renamed type to obligationType ,Changed required licenseIds to optional , Added optional ObligationLevel obligationLevel in ObligationStatusInfo
> - Remove ProjectTodo and UI changes for Obligation and ProjectTodo 

> * Which issue is this pull request belonging to and how is it solving it? (*Refer to issue here*)
> * Did you add or update any new dependencies that are required for your change?

### How To Test?
> Create Admin -> Project/Component/Organisation Obligation
> Obligations created should be reflected in Obligation tab of Project.
> Edit Obligation 
> Add/remove release with 1 approved attachment in Project, License Obligation chnages should be reflected.

> Have you implemented any additional tests? - No

### Checklist
Must:

Signed-off-by: Jaideep Palit <jaideep.palit@siemens.com>
